sonic-utilities-tests/sku_create_test.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import os
+import pytest
+import swsssdk
```

Comment:
@madhanmellanox please remove unused imports

Reply:
Addressed in next commit.